### DISCUSSION TITLE
SUPERCRONIC_EXTRA_FLAGS purpose

### DISCUSSION THREAD

**Discussion by televisi** [borgmatic-collective/docker-borgmatic#297] [General]
Could someone please assist me in understanding the purpose of SUPERCRONIC_EXTRA_FLAGS?
I noticed it was mentioned in #203 but unsure how to use / the purpose of it.

Currently shown on my log:

```
INFO[2024-01-30T22:04:35+11:00] crontab is valid                             
The variable SUPERCRONIC_EXTRA_FLAGS is empty, starting normally
INFO[2024-01-30T22:04:35+11:00] read crontab: /etc/borgmatic.d/crontab.txt 
```

Thanks

**Comment by grantbevis** [comment]
It's so you can pass extra flags to the supercronic binary https://github.com/borgmatic-collective/docker-borgmatic/blob/42e312f0fba61e0fa328e7334c483c92e5f8b5de/entry.sh#L88

Additional flags like `-debug` and so on

**Comment by televisi** [comment]
Got it, thanks!
